tests/e2e-evm/hardhat.config.ts

```diff
@@ -0,0 +1,68 @@
+import { HardhatUserConfig, extendEnvironment } from "hardhat/config";
+import "@nomicfoundation/hardhat-viem";
+import { parseEther } from "viem";
+import { extendViem } from "./test/extend";
+import chai from "chai";
```

Comment:
ohh we gotta chai guy, eh?! 😆

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExZGx3ejZ1ajg4bmwxYmthZm44YWU0Nnl5dzc2Znozc3k3cjl1dXZwNCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/lIOuN8T4na9AdmaRIf/giphy.gif)

<meaningless comment, please ignore>

Reply:
🤣